hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/metadata/SparkHoodieBackedTableMetadataWriter.java

```diff
@@ -169,58 +157,45 @@ public void deletePartitions(String instantTime, List<MetadataPartitionType> par
     writeClient.deletePartitions(partitionsToDrop, instantTime);
   }
 
+  @Override
+  protected HoodieData<HoodieRecord> getExpressionIndexUpdates(Option<HoodiePairData<String, List<List<HoodieColumnRangeMetadata<Comparable>>>>> partitionRangeMetadataPairOpt,
```

Comment:
I don't see `partitionRangeMetadataPairOpt` getting used.

Reply:
Removed it. This was used in the older approach.